ilastik/widgets/exportObjectInfoDialog.py

```diff
@@ -256,15 +258,12 @@ def validate_before_exit(self):
             self.ui.toolBox.setCurrentIndex(0)
             return
         else:
-            path = str(self.ui.exportPath.text())
-            if not self.is_valid_path(path):
+            path = PathComponents(str(self.ui.exportPath.text()))
+            if not path.extension not in ALLOWED_EXTENSIONS:
                 title = "Warning"
                 text = "No file extension or invalid file extension ( %s )\nAllowed: %s"
-                match = path.rsplit(".", 1)
-                if len(match) == 1:
-                    ext = "<none>"
-                else:
-                    ext = match[1]
+
+                ext = path.ext or "<no file extension>"
                 text %= (ext, ", ".join(ALLOWED_EXTENSIONS))
```

Comment:
This is a very... creative way to format a string. We probably should change this to an f-string.

Reply:
pushed the wrong branch